style.css

```diff
@@ -0,0 +1,99 @@
+*,
+::before,
+::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.navigation {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 3em;
+  background-color: #fff;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 0 24px;
+  z-index: 20;
+}
+
+.burger_menu {
+  display: block;
+  width: 24px;
+  height: 16px;
+}
+
+.bar {
+  display: block;
+  width: 24px;
+  height: 3px;
+  border-radius: 8px;
+  margin: 5px auto;
+  background-color: #000;
+  -webkit-transition: all 0.5s ease-in-out;
+  transition: all 0.5s ease-in-out;
+}
+
+.main_page {
+  position: relative;
+  top: 3em;
+  width: 100vw;
+  height: 100vh;
+  padding-top: 48px;
+  align-content: center;
+  background-image: url('Assets/Illustrations/headline illustration.png');
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] Please 🙏🏻 use the `flexbox` property for your section container because `flex` helps us to align the children's elements properly inside the container with equal spacing. I would also suggest you remove the `position` from your code because dealing with the position property changes the behaviour of the flow of the website.